packages/desktop-client/src/components/modals/CreateLocalAccountModal.tsx

```diff
@@ -48,13 +70,20 @@ export function CreateLocalAccountModal({
             onSubmit={async event => {
               event.preventDefault();
 
-              const nameError = !name;
-              setNameError(nameError);
+              const nameError = validateName(name, accounts);
 
+              if (nameError.error) {
+                setNameError(nameError);
+              } else {
+                setNameError({ error: false, message: '' });
+              }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could simplify this further

```suggestion
              setNameError(nameError);
```

Reply:
I can take a look at that, I think when I tried that maybe it needed to get un-set, if not, I'm not sure why I didn't do that in the first place haha